src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final int COLOR_NUMBER = 4;
```

Comment:
It's better to use Color.values().length instead of hardcoded 4. In case you add some colors to enum, you could forget to increase the constant COLOR_NUMBER.